packages/dialtone-vue2/components/hovercard/hovercard.test.js

```diff
@@ -0,0 +1,147 @@
+import { mount } from '@vue/test-utils';
+import DtHovercard from './hovercard.vue';
+
+const MOCK_DEFAULT_SLOT_MESSAGE = 'Message';
+const MOCK_HEADER_CONTENT = 'Hovercard Title';
+const MOCK_FOOTER_CONTENT = 'Hovercard Footer';
+
+const baseProps = { id: 'hovercard-1' };
+const baseAttrs = {};
+const baseSlots = {
+  content: MOCK_DEFAULT_SLOT_MESSAGE,
+  headerContent: MOCK_HEADER_CONTENT,
+  footerContent: MOCK_FOOTER_CONTENT,
+};
+
+const baseScopedSlots = {
+  anchor: '<template #anchor="{ attrs }">' +
+          '<button data-qa="dt-button" v-bind="attrs">Hover me</button>' +
+          '</template>',
+};
+
+let mockProps = {};
+let mockAttrs = {};
+let mockSlots = {};
+
+describe('DtHovercard Tests', () => {
+  let anchor;
+  let hovercardWindow;
+  let content;
+  let button;
+  let wrapper;
+
+  const updateWrapper = () => {
+    wrapper = mount(DtHovercard, {
+      propsData: { ...baseProps, ...mockProps },
+      attrs: { ...baseAttrs, ...mockAttrs },
+      slots: { ...baseSlots, ...mockSlots },
+      scopedSlots: { ...baseScopedSlots },
+      global: {
+        stubs: {
+          transition: false,
+        },
+      },
+      attachTo: document.body,
+    });
+
+    hovercardWindow = wrapper.find('[data-qa="dt-hovercard"]');
+    anchor = wrapper.find('[data-qa="dt-hovercard-anchor"]');
+    content = wrapper.find('[data-qa="dt-popover-content"]');
+    button = wrapper.find('[data-qa="dt-button"]');
+  };
+
+  beforeEach(() => {
+    vi.useFakeTimers();
+    vi.runAllTimers();
+    updateWrapper();
+  });
+
+  afterEach(() => {
+    vi.useRealTimers();
+    vi.clearAllTimers();
```

Comment:
Not so sure about using timers in this tests....
If you remove all of them the tests still passes, It would be necessary to debug this a little bit to see if it indeed affecting something. 
If it is not, maybe it is trowing a fake check.

Reply:
This is quite weird, without the timers, the tests do pass, but the popover `isOpen` is never set. So it's giving a false positive.
With the timers, this log does show:
<img width="429" alt="image" src="https://github.com/dialpad/dialtone/assets/24460973/009788ef-1b69-41fc-9f1a-88d7037f9a0c">

Will investigate, thanks for catching this!

Reply:
Fixed in latest changes